core/src/main/kotlin/fr/sncf/osrd/envelope_sim_infra/MRSP.kt

```diff
@@ -72,36 +74,39 @@ fun computeMRSP(
     trainTag: String?,
     temporarySpeedLimitManager: TemporarySpeedLimitManager?,
     safetySpeedRanges: DistanceRangeMap<Speed>? = null,
+    ignoreInfraSpeedLimits: Boolean = false,
```

Comment:
The parameter isn't set.

It should be done in `StandaloneSimulation.kt:71`. And it should also be added to the other version of the method which takes a rolling stock as parameter (this file, line 35).